.github/workflows/test.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v1
     - name: Test as root, without cgo, and with busybox
-      run: docker run -v=$PWD:/pwd golang:1.13-alpine3.10 sh -c 'export CGO_ENABLED=0; cd /pwd && go test ./...'
+      run: docker run -v=$PWD:/pwd golang:1.13-alpine3.11 sh -c 'export CGO_ENABLED=0; cd /pwd && go test ./...'
```

Comment:
While I know this is yaml, I still personally prefer to quote and brace shell stuffs.
e.g. -v="${PWD}:/pwd"
 actually, isn't $(pwd) preferred over $PWD?

in any case, also i think it's better to just set the 'workdir'.

```yaml
run: docker run --volume "$(pwd):/workdir" --workdir "/workdir" golang:1.13-alpine3.11 sh -c 'export CGO_ENABLED=0; go test ./...'
```

hmm, i just noticed you only bumped the version here :) leaving this for the future then

Reply:
In general I agree with you. Hadolint is a nice tool that can check these sorts of things, and some checker should be in one's automated pre-commit checks if there are Dockerfiles in the repo. But as you say I only bump the alpine version here so I don't think it's appropriate to start muddling with everything else here too.